ch.sbb.scion.rcp.microfrontend/src/ch/sbb/scion/rcp/microfrontend/browser/RxJsObservable.java

```diff
@@ -23,17 +23,17 @@
  */
 public class RxJsObservable<T> {
 
-  private CompletableFuture<Browser> whenBrowser;
+  private CompletableFuture<BrowserView> whenBrowser;
   private Type clazz;
   private String rxjsObservableIIFE;
 
-  public RxJsObservable(CompletableFuture<Browser> browser, String rxjsObservableIIFE, Type clazz) {
+  public RxJsObservable(final CompletableFuture<BrowserView> browser, final String rxjsObservableIIFE, final Type clazz) {

```

Comment:
It is probably worth documenting why the `clazz` parameter is not just `Class<?>`.

Reply:
The `rxjsObservableIIFE` parameter is also mysterious, especially because of the abbreviation. I suspect it's probably a piece of project knowledge. But when reviewed in a scope of a subsystem it's probably worth documenting.

Reply:
@alexander-yevsyukov PTAL at the Javadoc.